vnext/ReactWindowsCore/Modules/PlatformConstantsModule.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
>L [](start = 26, length = 1)

nit: must be lower case 'l'

Reply:
I'll do a followup to replace copyright headers. We have a bunch using Facebook we shouldn't as well.